nextcord/user.py

```diff
@@ -247,11 +247,14 @@ def created_at(self) -> datetime:
     def display_name(self) -> str:
         """:class:`str`: Returns the user's display name.
 
-        For regular users this is just their username, but
-        if they have a guild specific nickname then that
-        is returned instead.
+        Returns the users displayed name.
+        So whatever is the first thing to exist from the following priority list:
```

Comment:
Same here
```suggestion
        This will return the name using the following hierachy:
```
Alternatively, something along the lines of `Returns the user's global name if any, else the username will be returned`